recipe/meta.yaml

```diff
@@ -23,30 +22,34 @@ requirements:
     - pip
     - python
     - flit-core >=3.2,<4
-    - setuptools
-    - wheel
   run:
     - python
-    - numpy >=1.17,!=1.24.0
-    # matplotlib 3.6.1 incompatible with latest master for 0.12.0
-    # https://github.com/mwaskom/seaborn/issues/3072
-    - matplotlib-base >=3.1,!=3.6.1
-    - pandas >=0.25
-    - typing_extensions  # [py<38]
+    - numpy >=1.20,!=1.24.0
+    - matplotlib-base >=3.4,!=3.6.1
+    - pandas >=1.2
   run_constrained:
-    - scipy >=1.3
-    - statsmodels >=0.10
+    - scipy >=1.7
+    - statsmodels >=0.12
 
 test:
+  source_files:
+    - tests
+    - pyproject.toml
   imports:
     - seaborn
     - seaborn.colors
     - seaborn.external
     - seaborn.objects
   commands:
     - pip check
+    - pytest -n auto tests -vv
   requires:
     - pip
+    - pytest
+    - pytest-xdist
+    - scipy
+    - statsmodels
+    - matplotlib
```

Comment:
```suggestion
```
`matplotlib-base` is already available in `run`

Reply:
For some reason, during testing, it couldn't find it so it would resort to `tk` as its backend, but that was causing errors, so adding `matplotlib` to the `test` section ensured `qt`'s backend was used instead.